dropshot/src/handler.rs

```diff
@@ -570,7 +570,7 @@ where
  */
 trait GetMetadata {
     fn metadata(loc: ApiEndpointParameterLocation)
-        -> Vec<ApiEndpointParameter>;
+    -> Vec<ApiEndpointParameter>;
```

Comment:
this.... seems like a bug to me

Reply:
This is the change I'm not a fan of; agreed that it probably should be filed as a bug.

Reply:
https://github.com/rust-lang/rustfmt/issues/4366

Reply:
fixed: https://github.com/rust-lang/rustfmt/issues/4366